front/app/component-library/utils/styleUtils.ts

```diff
@@ -453,6 +455,12 @@ export function getTheme(tenant: any = null): MainThemeProps {
       ...colors,
       tenantPrimary: core ? core.color_main : '#ef0071',
       tenantSecondary: core ? core.color_secondary : '#000000',
+      tenantPrimaryLighten75: core
+        ? transparentize(0.75, core.color_main)
+        : transparentize(0.75, '#ef0071'),
+      tenantPrimaryLighten95: core
+        ? transparentize(0.95, core.color_main)
+        : transparentize(0.95, '#ef0071'),
       tenantText: core ? core.color_text : '#333333',
```

Comment:
Added as discussed [here](https://go-vocal.slack.com/archives/CQAGMHKT3/p1717583294555739?thread_ts=1717416779.479429&cid=CQAGMHKT3) in Slack.